docs/reference/submission_schema_v1.rst

```diff
@@ -0,0 +1,8 @@
+Version 1
+=========
+
+This document describes the schema of the ``test_submission_schema/v1.json`` files as part of the submission. To get the latest JSON schema file, go to the Test Submission Schema repository.
+        
+.. jsonschema:: ../../test_submission_schema/v1.json
```

Comment:
You can change this file to Markdown and use the MyST syntax for extension, so all doc files will be consistent. The following block should work in `.md`:

```
```{jsonschema} ../../test_submission_schema/v1.json
    :lift_definitions:
    :auto_reference:
```

Reply:
Changed in ff0943e